### PR TITLE
gh-101100: Fix Sphinx warnings in `whatsnew/3.2.rst`

### PR DESCRIPTION
**Pull request by hugovk**
Fix 49 warnings:

```
Doc/whatsnew/3.2.rst:345: WARNING: py:class reference target not found: importlib.abc.PyLoader
Doc/whatsnew/3.2.rst:345: WARNING: py:class reference target not found: importlib.abc.PyPycLoader
Doc/whatsnew/3.2.rst:400: WARNING: py:func reference target not found: start_response
Doc/whatsnew/3.2.rst:418: WARNING: py:meth reference target not found: write
Doc/whatsnew/3.2.rst:418: WARNING: py:func reference target not found: start_response
Doc/whatsnew/3.2.rst:500: WARNING: py:meth reference target not found: __getattr__
Doc/whatsnew/3.2.rst:500: WARNING: py:meth reference target not found: __getattribute__
Doc/whatsnew/3.2.rst:621: WARNING: py:class reference target not found: collections.Sequence
Doc/whatsnew/3.2.rst:723: WARNING: py:mod reference target not found: xml.etree.cElementTree
Doc/whatsnew/3.2.rst:1010: WARNING: py:data reference target not found: time.accept2dyear
Doc/whatsnew/3.2.rst:1015: WARNING: py:data reference target not found: time.accept2dyear
Doc/whatsnew/3.2.rst:1034: WARNING: py:data reference target not found: time.accept2dyear
Doc/whatsnew/3.2.rst:1151: WARNING: py:meth reference target not found: __repr__
Doc/whatsnew/3.2.rst:1157: WARNING: py:meth reference target not found: __repr__
Doc/whatsnew/3.2.rst:1157: WARNING: py:meth reference target not found: __repr__
Doc/whatsnew/3.2.rst:1338: WARNING: py:attr reference target not found: Context.clamp
Doc/whatsnew/3.2.rst:1431: WARNING: py:class reference target not found: zipfile.ZipExtFile
Doc/whatsnew/3.2.rst:1599: WARNING: py:attr reference target not found: sqlite3.Connection.in_transit
Doc/whatsnew/3.2.rst:1646: WARNING: py:func reference target not found: ssl.match_hostname
Doc/whatsnew/3.2.rst:1650: WARNING: py:func reference target not found: ssl.wrap_socket
Doc/whatsnew/3.2.rst:1761: WARNING: py:class reference target not found: unittest.case.TestCase
Doc/whatsnew/3.2.rst:1799: WARNING: py:meth reference target not found: unittest.TestCase.assertRegexpMatches
Doc/whatsnew/3.2.rst:1816: WARNING: py:meth reference target not found: assert_
Doc/whatsnew/3.2.rst:1817: WARNING: py:meth reference target not found: assertEquals
Doc/whatsnew/3.2.rst:1818: WARNING: py:meth reference target not found: assertNotEquals
Doc/whatsnew/3.2.rst:1819: WARNING: py:meth reference target not found: assertAlmostEquals
Doc/whatsnew/3.2.rst:1820: WARNING: py:meth reference target not found: assertNotAlmostEquals
Doc/whatsnew/3.2.rst:1827: WARNING: py:meth reference target not found: unittest.TestCase.assertDictContainsSubset
Doc/whatsnew/3.2.rst:2000: WARNING: py:meth reference target not found: get
Doc/whatsnew/3.2.rst:2000: WARNING: py:meth reference target not found: setdefault
Doc/whatsnew/3.2.rst:2121: WARNING: py:class reference target not found: Pdb
Doc/whatsnew/3.2.rst:2397: WARNING: py:meth reference target not found: split
Doc/whatsnew/3.2.rst:2397: WARNING: py:meth reference target not found: rsplit
Doc/whatsnew/3.2.rst:2397: WARNING: py:meth reference target not found: splitlines
Doc/whatsnew/3.2.rst:2397: WARNING: py:meth reference target not found: replace
Doc/whatsnew/3.2.rst:2397: WARNING: py:meth reference target not found: rfind
Doc/whatsnew/3.2.rst:2397: WARNING: py:meth reference target not found: rindex
Doc/whatsnew/3.2.rst:2397: WARNING: py:meth reference target not found: rsplit
Doc/whatsnew/3.2.rst:2397: WARNING: py:meth reference target not found: rpartition
Doc/whatsnew/3.2.rst:2411: WARNING: py:meth reference target not found: array.repeat
Doc/whatsnew/3.2.rst:2411: WARNING: py:class reference target not found: BaseHTTPRequestHandler
Doc/whatsnew/3.2.rst:2565: WARNING: c:macro reference target not found: Py_VA_COPY
Doc/whatsnew/3.2.rst:2569: WARNING: c:func reference target not found: PySys_SetArgvEx
Doc/whatsnew/3.2.rst:2653: WARNING: py:meth reference target not found: array.tostring
Doc/whatsnew/3.2.rst:2653: WARNING: py:meth reference target not found: array.fromstring
Doc/whatsnew/3.2.rst:2653: WARNING: py:meth reference target not found: array.tobytes
Doc/whatsnew/3.2.rst:2653: WARNING: py:meth reference target not found: array.frombytes
Doc/whatsnew/3.2.rst:2667: WARNING: py:func reference target not found: sys.setfilesystemencoding
Doc/whatsnew/3.2.rst:2675: WARNING: py:func reference target not found: string.maketrans
```

* Issue: gh-101100

----
📚 Documentation preview 📚: https://cpython-previews--115580.org.readthedocs.build/

### CODE REVIEW

File: Doc/whatsnew/3.2.rst [hunk 1]
```diff
@@ -1336,7 +1336,7 @@ Decimal('1.100000000000000088817841970012523233890533447265625')
 Fraction(2476979795053773, 2251799813685248)
 
 Another useful change for the :mod:`decimal` module is that the
-:attr:`Context.clamp` attribute is now public.  This is useful in creating
+:attr:`Context.clamp <decimal.Context>` attribute is now public.  This is useful in creating
```
Comment (AlexWaygood) [inline Doc/whatsnew/3.2.rst]: Public... but apparently still undocumented? Should this maybe be left as is, and added to this list for now?

https://github.com/python/cpython/blob/318f2190bc93796008b0a4241243b0851b418436/Doc/conf.py#L273-L279

**Review by AlexWaygood — APPROVED** [review]
Nice! Just one question:

File: Doc/whatsnew/3.2.rst (hunk 1 above)
Comment (hugovk) [reply Doc/whatsnew/3.2.rst]: Sure, added!

**Review by AlexWaygood — APPROVED** [review]
Thanks!

**Pull request merged**